tools/configgen/pkg/testgrid.go

```diff
@@ -25,7 +25,7 @@ import (
 	"sort"
 	"strings"
 
-	"knative.dev/test-infra/pkg/testgrid"
+	"knative.dev/test-infra/tools/configgen/pkg/testgrid"
```

Comment:
?

Reply:
The configgen is using the `./pkg/testgrid` package. I'm also removing the whole `./pkg` package, so moving those two files to the `configgen` directory makes sense to me.